### ISSUE TITLE
TextContentRenderer throws NullPointerException with nested list

### ISSUE THREAD

**Issue by jaystrictor** [atlassian/commonmark-java#90]
Trying to render a nested list, like
```
1. 1.
```
with the TextContentRenderer throws a NullPointerException:

```
Exception in thread "main" java.lang.NullPointerException
	at org.commonmark.renderer.text.CoreTextContentNodeRenderer.visit(CoreTextContentNodeRenderer.java:154)
	at org.commonmark.node.ListItem.accept(ListItem.java:7)
	at org.commonmark.renderer.text.CoreTextContentNodeRenderer.render(CoreTextContentNodeRenderer.java:56)
	at org.commonmark.internal.renderer.NodeRendererMap.render(NodeRendererMap.java:23)
	at org.commonmark.renderer.text.TextContentRenderer$RendererContext.render(TextContentRenderer.java:146)
	at org.commonmark.renderer.text.CoreTextContentNodeRenderer.visitChildren(CoreTextContentNodeRenderer.java:198)
	at org.commonmark.renderer.text.CoreTextContentNodeRenderer.visit(CoreTextContentNodeRenderer.java:168)
	at org.commonmark.node.OrderedList.accept(OrderedList.java:10)
	at org.commonmark.renderer.text.CoreTextContentNodeRenderer.render(CoreTextContentNodeRenderer.java:56)
	at org.commonmark.internal.renderer.NodeRendererMap.render(NodeRendererMap.java:23)
	at org.commonmark.renderer.text.TextContentRenderer$RendererContext.render(TextContentRenderer.java:146)
	at org.commonmark.renderer.text.CoreTextContentNodeRenderer.visitChildren(CoreTextContentNodeRenderer.java:198)
	at org.commonmark.renderer.text.CoreTextContentNodeRenderer.visit(CoreTextContentNodeRenderer.java:62)
	at org.commonmark.node.Document.accept(Document.java:7)
	at org.commonmark.renderer.text.CoreTextContentNodeRenderer.render(CoreTextContentNodeRenderer.java:56)
	at org.commonmark.internal.renderer.NodeRendererMap.render(NodeRendererMap.java:23)
	at org.commonmark.renderer.text.TextContentRenderer$RendererContext.render(TextContentRenderer.java:146)
	at org.commonmark.renderer.text.TextContentRenderer.render(TextContentRenderer.java:44)
	at org.commonmark.renderer.text.TextContentRenderer.render(TextContentRenderer.java:50)
```

**Comment by JinneeJ** [comment]
@jaystrictor Thanks for the report! 

@robinst Actually I'm about to open PR with some fixes for `TextContentRenderer` soon.

**Comment by robinst** [comment]
Nice, thanks Julia!

**Comment by robinst** [comment]
Fixed via #92.

**Comment by robinst** [comment]
Released in 0.10.0 now: https://github.com/atlassian/commonmark-java/blob/master/CHANGELOG.md#0100---2017-09-14
